windows/ReactTestApp/MainPage.cpp

```diff
@@ -6,27 +6,63 @@
 #include "MainPage.g.cpp"
 #include "Manifest.h"
 
-using winrt::Windows::Foundation::Collections::IVector;
+using namespace winrt::Windows::UI::Xaml::Controls;
 
 namespace winrt::ReactTestApp::implementation
 {
     MainPage::MainPage()
     {
         InitializeComponent();
+
+        auto menuItems = MenuFlyout().Items();
```

Comment:
Reference the array:
```suggestion
        auto &menuItems = MenuFlyout().Items();
```

Reply:
Is this the same as above where we can modify the copy and it still works?

Reply:
Yes